packages/taquito-rpc/src/types.ts

```diff
@@ -58,9 +50,9 @@ interface Frozenbalancebycycle {
 
 export type BigMapKey = { key: { [key: string]: string }; type: { prim: string } };
 
-export type ContractResponse = ContractResponse004 | ContractResponse005;
+export type ContractResponse = ContractResponse005;
```

Comment:
Is `ContractResponse005` copatible with Carthagnet `006`? 
Would it be reasonable/useful (to the reader of the code) to alias `ContractResponse005` as `ContractResponse006`? It would have no functional value for the code but might make the code more literate. I don't feel strongly about this, it's for discussion.

Reply:
I agree, I remove all suffix for RPC types as there should be no difference between 005 and 006 response now